parachain-template/node/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+pub mod chain_spec;
+mod rpc;
+pub mod service;
```

Comment:
I think this file is not used anywhere, for Substrate's `node-template` I removed it [here](https://github.com/paritytech/substrate/pull/9667).